doc/man3/EVP_EncryptInit.pod

```diff
@@ -100,7 +100,6 @@ EVP_enc_null
  int EVP_CIPHER_nid(const EVP_CIPHER *e);
  int EVP_CIPHER_block_size(const EVP_CIPHER *e);
  int EVP_CIPHER_key_length(const EVP_CIPHER *e)
- int EVP_CIPHER_key_length(const EVP_CIPHER *e);
```

Comment:
Probably it would be better to remove the one without the ; on the end!

Reply:
Indeed thanks!